ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -208,16 +209,15 @@ export const WorkspaceAbout = fp.flow(
 
     loadUserRoles(workspace: WorkspaceData) {
       this.setState({ workspaceUserRoles: [] });
-      workspacesApi()
-        .getFirecloudWorkspaceUserRoles(workspace.namespace, workspace.id)
-        .then((resp) => {
-          this.setState({
-            workspaceUserRoles: fp.sortBy('familyName', resp.items),
-          });
-        })
-        .catch((error) => {
-          console.error(error);
-        });
+      fetchWithErrorModal(() =>
```

Comment:
the simplest way to user the new `fetchWithErrorModal`: a default error popup will appear.  See errors and errors.spec for more examples